src/patternfly/components/Card/examples/Card.md

```diff
@@ -381,7 +413,9 @@ A card is a generic rectangular container that can be used to build other compon
 | `.pf-m-no-fill` | `.pf-c-card__body` | Sets a `.pf-c-card__body` to not fill the available space in `.pf-c-card`. `.pf-m-no-fill` can be added to multiple card bodies. |
 | `.pf-m-hoverable` | `.pf-c-card` | Modifies the card to include hover styles on `:hover`. |
 | `.pf-m-selectable` | `.pf-c-card` | Modifies a selectable card so that it is selectable. |
-| `.pf-m-selected` | `.pf-c-card.pf-m-selectable` | Modifies a selectable card for the selected state. |
-| `.pf-m-flat` | `.pf-c-card` | Modifies the card to have a border instead of a shadow. `.pf-m-flat` is for use in layouts where cards are against a white background. |
+| `.pf-m-selected` | `.pf-c-card.pf-m-selectable` | Modifies a selePlain
+| `.pf-m-flat` | `.pf-c-card` | Modifies the card to have a border instead of a shadow. `.pf-plain` is for use in layouts where cards are against a white background. |
```

Comment:
I think this should stay `.pf-m-flat`? It could also just say "Flat". Or "For use in layouts where cards are against a white background."